gulpfile.js

```diff
@@ -155,33 +155,57 @@ var findTag = function(stringContainsTag) {
     return stringContainsTag.match(regex)[1];
 }
 
-var transcodeReadme = function(inputFile, outputFile, tag) {
-    const lines = fs.readFileSync(inputFile).toString('utf8').split("\n")
+var transcribeReadme = function(inputFile, outputFile, tag) {
+    const lines = fs.readFileSync(inputFile).toString('utf8').split("\n");
     const outputHeader = `# Resource
 > see https://aka.ms/autorest
 This is the AutoRest configuration file for Resource.
 ---
 `;
-    const outputLines = []
+
+    const outputLines = [];
+
+    // transcribe common config
+    let commonFound = false;
+    const commonFields = ['title', 'description', 'openapi-type']
```

Comment:
Copy these to tmp md.

```
\``` yaml
title: WebSiteManagementClient
description: WebSite Management Client
openapi-type: arm
\```
```